app/views/directives/_service-integration.html

```diff
@@ -0,0 +1,19 @@
+<div class="service-integration" ng-if="$ctrl.doRender">
+  <div class="row" ng-class="{'active-integration': $ctrl.getState() == 1,
+                              'inactive-integration': $ctrl.getState() == 0,
+                              'unknown-integration': $ctrl.getState() == -1}">
+    <span class="image-icon">
+      <img ng-src="{{ $ctrl.integrationData.imageData.imageUrl }}"/>
+    </span>
+    <span class="description">
+      <h4>
+        {{$ctrl.integrationData.name}}
+        <span class="del-integration" ng-if="$ctrl.getState() == 1">-</span>
```

Comment:
I have seen digest errors when functions are used on directives like this (usually when in a `ng-repeat`)
We are only displaying one integration currently but basing this on a property is a better idea.
We can discuss if you want.